protocol/bin/js.js

```diff
@@ -36,9 +36,15 @@ function addEnums (prot, json) {
 }
 
 function write () {
-  var s = fs.createWriteStream('js/keybase_v1.js')
-  s.write('export default ' + JSON.stringify(protocols, null, 2).replace(/\"/g, '\'') + '\n')
-  s.close()
+  var stream = fs.createWriteStream('js/keybase_v1.js')
```

Comment:
its an error to export a module like 
```
export default const { 
a: 1,
b: 2}
```

then do

```
import {a} from './blah'
```

This worked in babel 5 but won't in babel 6